bro_log_transport.sh

```diff
@@ -192,10 +188,22 @@ if ! can_ssh "$aih_location" "-o" 'PasswordAuthentication=no' $extra_ssh_params
 	fi
 fi
 
-#What local directory holds the bro logs?
+#What local directory holds the bro/zeek logs?
 #Make sure the directory ends in a "/".
 if [ -z "$local_tld" ]; then
-	if [ -d /storage/bro/logs/ ]; then				#Custom
+	# Check for zeek paths first
+	if [ -d /storage/zeek/logs/ ]; then				#Custom
+		local_tld='/storage/zeek/logs/'
+	elif [ -d /opt/zeek/logs/ ]; then				#Zeek as installed by Rita
+		local_tld='/opt/zeek/logs/'
+	elif [ -d /usr/local/zeek/logs/ ]; then				#Zeek default
+		local_tld='/usr/local/zeek/logs/'
+	elif [ -d /var/lib/docker/volumes/var_log_zeek/_data/ ]; then	#Blue vector
+		local_tld='/var/lib/docker/volumes/var_log_zeek/_data/'
```

Comment:
@ethack do you remember what type of Bro install the `blue vector` log location is for? I'm not sure if we need to change this to zeek or how to test it. Any guidance would be appreciated. Thanks.

Reply:
I think we can just leave it as-is for now without worrying about migrating it to Zeek.